depmapomics/qc/rna.py

```diff
@@ -4,41 +4,59 @@
 from genepy import rna
 from depmapomics import terra
 import numpy as np
-from depmapomics.config import (RNASEQC_THRESHOLDS_FAILED,
-                                RNASEQC_THRESHOLDS_LOWQUAL)
-
-
-def plot_rnaseqc_results(workspace, samplelist, output_path='data/rna_qcs/', 
-                         qcname="rnaseqc2_metrics", save=True):
-  """
+from depmapomics.config import RNASEQC_THRESHOLDS_FAILED, RNASEQC_THRESHOLDS_LOWQUAL
+
+
+def plot_rnaseqc_results(
```

Comment:
some files in QC are pytests and some are functions that are used in the release (different to QC the data, which is something we do as part of the release. and QC the pipeline which we do as part of our updates of the code)

Reply:
@5im1z